main.py

```diff
@@ -43,9 +43,9 @@ def main() -> None:
         pass
 
     if args.info is not None:
-        print_game_info(args.info)
+        print_game_info(args.info.lstrip('MatchID='))
```

Comment:
I think it might be good to just create a function like `normalize_match_id_string` that you can then call anywhere to do the replace/other possible future changes. That way it's all in one place. Maybe just a lose function in data_controller.py or a new file that just contains utility functions. I'm already thinking DataController could be split into two, and this might belong in one of those halves.